iceoryx_binding_c/test/moduletests/test_types_storage_size.hpp

```diff
@@ -34,6 +34,76 @@ extern "C" {
 
 #include <iostream>
 
+inline void verifyStorageSizeCalculationForListener()
```

Comment:
Those tests are a bit odd. Could we not split them up for the different integer sizes? E.g. `verifyStorageSizeCalculationForListenerUInt{8,16, ..}`

Reply:
Here I am testing if the size calculation function is correct by picking multiple values which have to map to the underlying linear function.

What would be the benefit of using different integer sizes? `sizeof` always returns an `uint64_t` and the `ListenerImpl` capacity is also always the same integer size?